R/estimate_infections.R

```diff
@@ -455,26 +455,15 @@ fit_model_with_nuts <- function(args, future = FALSE, max_execution_time = Inf,
     }
   }
 
-  if (!future) {
-    fit <- fit_chain(1,
-      stan_args = args, max_time = max_execution_time,
-      catch = !id %in% c("estimate_infections", "epinow")
-    )
-    if (stuck_chains > 0) {
-      fit <- NULL
-    }
-    if (is.null(fit)) {
-      rlang::abort("model fitting was timed out or failed")
-    }
-  } else {
+  if (future) {
     chains <- args$chains
     args$chains <- 1
     args$cores <- 1
     fits <- future.apply::future_lapply(1:chains, fit_chain,
-      stan_args = args,
-      max_time = max_execution_time,
-      catch = TRUE,
-      future.seed = TRUE
+                                        stan_args = args,
+                                        max_time = max_execution_time,
+                                        catch = TRUE,
+                                        future.seed = TRUE
```

Comment:
This is deviating from our standard indenting style in the package (two spaces).

Reply:
This has been resolved in 841d34068d7b23f2979874fe0dd201da4e5c0792.